index-pl.html

```diff
@@ -271,23 +271,23 @@ <h2>From People to The Population</h2>
       <div class="row big-small-small">
 
         <div class="column onehundredpx">
-          <div class="center">Disease Transmission Route</div>
+          <div class="center">Droga Rozprzestrzeniania</div>
           <anim :mask1="false" :mask2="false"></anim>
           <anim :mask1="true" :mask2="false" :eout="0.5"></anim>
           <anim :mask1="false" :mask2="true" :ein="0.5"></anim>
           <anim :mask1="true" :mask2="true" :eout="0.5" :ein="0.5"></anim>
         </div>
 
         <div class="column onehundredpx">
-          <div class="center">Drop in Disease Transmission</div>
+          <div class="center">Spadek w Rozprzestrzenianiu</div>
           <div class="center"><span class="red">0%</span></div>
           <div class="center"><span class="orange">50%</span></div>
           <div class="center"><span class="orange">50%</span></div>
           <div class="center"><span class="yellow">75%</span></div>
         </div>
 
         <div class="column onehundredpx">
-          <div class="center">Chance of This Route</div>
+          <div class="center">Przwdopodobieństwo Każdej z Dróg Rozprzestrzeniania</div>
```

Comment:
Proponuję z wielkiej litery pisać tylko piersze słowo i nie pisać wielkimi literami pozostałe rzeczowniki. Myślę, że wygląda to lepiej : )

Reply:
Przwdopodobieństwo - prawdopodobieństwo literówka

Reply:
> Proponuję z wielkiej litery pisać tylko piersze słowo i nie pisać wielkimi literami pozostałe rzeczowniki. Myślę, że wygląda to lepiej : )

To musimy sprawdzić w pozostałych częściach, bo chyba używałam wielkich liter.

Reply:
Pewnie będziemy jeszcze robić final ultimate review dla całości zaraz przed mergowaniem do oryginału : )